Examples/Server/docker/otel-collector-config.yaml

```diff
@@ -0,0 +1,26 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: "otel-collector:4317"
+
+processors:
+  batch: {}
```

Comment:
Can we skip the batching processor here since we're already using `OTelBatchSpanProcessor` to send batched spans to the collector?

Reply:
I _think_ the processor in the collector can do quite a bit more than can be done in the`OTelBatchSpanExporter` or the SDK more generally, and re-batching is a common documented pattern documented in the OTel spec[^1]:

> The OpenTelemetry Metrics data model is designed to support these transformations both inside an SDK as the data originates, or as a reprocessing stage inside the OpenTelemetry collector. These transformations are:
>
> 1. Temporal reaggregation: Metrics that are collected at a high-frequency can be re-aggregated into longer intervals, allowing low-resolution timeseries to be pre-calculated or used in place of the original metric data.
> 2. Spatial reaggregation: Metrics that are produced with unwanted attributes can be re-aggregated into metrics having fewer attributes.
> 3. Delta-to-Cumulative: Metrics that are input and output with Delta temporality unburden the client from keeping high-cardinality state. The use of deltas allows downstream services to bear the cost of conversion into cumulative timeseries, or to forego the cost and calculate rates directly.

It then goes on to provide example use-cases[^2], which show the composition of SDK _and_ Collector batching.

[^1]: https://opentelemetry.io/docs/specs/otel/metrics/data-model/#events--data-stream--timeseries
[^2]: https://opentelemetry.io/docs/specs/otel/metrics/data-model/#example-use-cases

With that being said, I'm happy to remove it and, if people want it, they can reach for it via the OTel/Collector docs themselves.

WDYT?

Reply:
Thanks for explaining and sharing the OTel Collector docs. However, for this example I think it adds unnecessary complexity to the collector config and I'd be in favor of removing it. Perhaps we could link to the OTel Collector docs for batch span processing from the `OTelBatchSpanProcessor` docs in the future, explaining how both may be used together.

Reply:
Done.